app/code/core/Mage/Log/Model/Log.php

```diff
@@ -58,6 +58,9 @@ protected function _construct()
         $this->_init('log/log');
     }
 
+    /**
+     * @return float|int
```

Comment:
why float? the config description is "Save Expired Items Days" and default value is "180"

Reply:
'Mage::getStoreConfig(self::XML_LOG_CLEAN_DAYS)' returns a 'string' ... multiplied its float.

Reply:
ok, so we need a casting to int here.
I propose we either skip this docbloc change, and open a new PR fixing the docblock and adding cast.

Reply:
created an issue for this change https://github.com/OpenMage/magento-lts/issues/949